shell/initialize/index.js

```diff
@@ -86,14 +99,22 @@ Object.defineProperty(Vue.prototype, '$nuxt', {
   configurable: true
 });
 
+// HTML Metadata manager for Vue.js
+// believe it can be removed
```

Comment:
this might one of the locations where a dupe meta header tag is from. i think we were going to remove one of the ways, this could be another. there's the hardcoded one in index.html as well